src/scripts/itemDiscuss/dimItemDiscussService.factory.js

```diff
@@ -0,0 +1,18 @@
+import angular from 'angular';
+
+angular.module('dimApp')
+  .factory('dimItemDiscussService', ItemDiscussService);
+
+function ItemDiscussService($rootScope) {
+  return {
+    dialogOpen: false,
+    addItemToDiscuss: addItemToDiscuss
+  };
```

Comment:
Agree, but I'm waiting on the UI polish to see if any messaging implementation changed.